### ISSUE TITLE
add UI for plot export

### ISSUE THREAD

**Issue by s-u** [att/rcloud#792]
Allow some way to do "Download as PDF" and similar for plots created in notebooks by replaying the corresponding display list in another device.

**Comment by gordonwoodhull** [comment]
This is in 1.3. Export to PDF still has not happened, just waiting on a minor refactoring of notebook.R and perhaps http.R. I'll move that part of the conversation into a new ticket.
